encoders/nlp/TextPaddlehubEncoder/Dockerfile

```diff
@@ -1,14 +1,13 @@
-FROM paddlepaddle/paddle
+FROM jinaai/jina
```

Comment:
should we use `devel`?

Reply:
sorry apparently the new cookiecutter puts `jinaai/jina` without devel so I may be wrong